packages/messaging/lib/index.d.ts

```diff
@@ -1012,6 +1088,12 @@ export namespace FirebaseMessagingTypes {
      * @param enabled A boolean value to enable or disable exporting of message delivery metrics to BigQuery.
      */
     setDeliveryMetricsExportToBigQuery(enabled: boolean): Promise<void>;
```

Comment:
I think we also need to export this same feature with the current "experimental" web method name?

https://github.com/firebase/firebase-js-sdk/blob/49ee786f2b022e65aef45693e1a8b546d889ec10/packages/messaging/src/api.ts#L184